jekyll/_includes/site/header.html

```diff
@@ -1,2 +1,4 @@
 <h1>PostingPro</h1>
 <p class="tagline">Build inclusive job postings that engage quality candidates!</p>
+<p>Either start a new job posting or build from an existing template. As you type, the text will be analyzed for grade reading level and problamatic phrases that might deter the right job seeker from applying. If the reading level is too high, it will be flagged by a red line.</p>
+<p><span class="bold">Keep the reading level at or below the required education for the position so that it is more inclusive.</span><br>For example, if the job does not require a college degree then the reading level should not be above 12th grade. Remember, the average adult’s reading level is 7th/8th grade. </p>
```

Comment:
Yes I did see that. Also there is no explanation for why it's linked or what you're supposed to do with the information on the Wikipedia page.

I also would prefer to keep them on PostingPro rather than have them leave the site. I can change the text as you suggested though.